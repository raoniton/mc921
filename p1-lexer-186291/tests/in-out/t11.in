class Main {
    public static void main(String[] args) {
        Main mainObj = new Main();
        int m = -9;
        int n = -4;
        print(mainObj.f(m, n));
    }

    public int f(int m, int n) {
        if (!(n >= 2)) {
            n = n - 2;
            return n;
        } else {
            m = m - 1;
            return m;
        }
    }
}