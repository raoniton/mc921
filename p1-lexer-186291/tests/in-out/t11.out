LexToken(CLASS,'class',1,0)
LexToken(ID,'Main',1,6)
LexToken(LBRACE,'{',1,11)
LexToken(PUBLIC,'public',2,17)
LexToken(STATIC,'static',2,24)
LexToken(VOID,'void',2,31)
LexToken(MAIN,'main',2,36)
LexToken(LPAREN,'(',2,40)
LexToken(STRING,'String',2,41)
LexToken(LBRACKET,'[',2,47)
LexToken(RBRACKET,']',2,48)
LexToken(ID,'args',2,50)
LexToken(RPAREN,')',2,54)
LexToken(LBRACE,'{',2,56)
LexToken(ID,'Main',3,66)
LexToken(ID,'mainObj',3,71)
LexToken(ASSIGN,'=',3,79)
LexToken(NEW,'new',3,81)
LexToken(ID,'Main',3,85)
LexToken(LPAREN,'(',3,89)
LexToken(RPAREN,')',3,90)
LexToken(SEMI,';',3,91)
LexToken(INT,'int',4,101)
LexToken(ID,'m',4,105)
LexToken(ASSIGN,'=',4,107)
LexToken(MINUS,'-',4,109)
LexToken(INT_LITERAL,'9',4,110)
LexToken(SEMI,';',4,111)
LexToken(INT,'int',5,121)
LexToken(ID,'n',5,125)
LexToken(ASSIGN,'=',5,127)
LexToken(MINUS,'-',5,129)
LexToken(INT_LITERAL,'4',5,130)
LexToken(SEMI,';',5,131)
LexToken(PRINT,'print',6,141)
LexToken(LPAREN,'(',6,146)
LexToken(ID,'mainObj',6,147)
LexToken(DOT,'.',6,154)
LexToken(ID,'f',6,155)
LexToken(LPAREN,'(',6,156)
LexToken(ID,'m',6,157)
LexToken(COMMA,',',6,158)
LexToken(ID,'n',6,160)
LexToken(RPAREN,')',6,161)
LexToken(RPAREN,')',6,162)
LexToken(SEMI,';',6,163)
LexToken(RBRACE,'}',7,169)
LexToken(PUBLIC,'public',9,176)
LexToken(INT,'int',9,183)
LexToken(ID,'f',9,187)
LexToken(LPAREN,'(',9,188)
LexToken(INT,'int',9,189)
LexToken(ID,'m',9,193)
LexToken(COMMA,',',9,194)
LexToken(INT,'int',9,196)
LexToken(ID,'n',9,200)
LexToken(RPAREN,')',9,201)
LexToken(LBRACE,'{',9,203)
LexToken(IF,'if',10,213)
LexToken(LPAREN,'(',10,216)
LexToken(NOT,'!',10,217)
LexToken(LPAREN,'(',10,218)
LexToken(ID,'n',10,219)
LexToken(GE,'>=',10,221)
LexToken(INT_LITERAL,'2',10,224)
LexToken(RPAREN,')',10,225)
LexToken(RPAREN,')',10,226)
LexToken(LBRACE,'{',10,228)
LexToken(ID,'n',11,242)
LexToken(ASSIGN,'=',11,244)
LexToken(ID,'n',11,246)
LexToken(MINUS,'-',11,248)
LexToken(INT_LITERAL,'2',11,250)
LexToken(SEMI,';',11,251)
LexToken(RETURN,'return',12,265)
LexToken(ID,'n',12,272)
LexToken(SEMI,';',12,273)
LexToken(RBRACE,'}',13,283)
LexToken(ELSE,'else',13,285)
LexToken(LBRACE,'{',13,290)
LexToken(ID,'m',14,304)
LexToken(ASSIGN,'=',14,306)
LexToken(ID,'m',14,308)
LexToken(MINUS,'-',14,310)
LexToken(INT_LITERAL,'1',14,312)
LexToken(SEMI,';',14,313)
LexToken(RETURN,'return',15,327)
LexToken(ID,'m',15,334)
LexToken(SEMI,';',15,335)
LexToken(RBRACE,'}',16,345)
LexToken(RBRACE,'}',17,351)
LexToken(RBRACE,'}',18,353)
